Last-Update: 2019-10-29
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: use external/system "varlink-go-interface-generator".

--- a/cmd/podman/varlink/generate.go
+++ b/cmd/podman/varlink/generate.go
@@ -1,3 +1,3 @@
 package iopodman
 
-//go:generate go run ../../../vendor/github.com/varlink/go/cmd/varlink-go-interface-generator/main.go io.podman.varlink
+//go:generate varlink-go-interface-generator io.podman.varlink
