Last-Update: 2019-06-11
Forwarded: not-needed
Bug-Upstream: https://github.com/containers/libpod/issues/3294
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: replace obsolete fork

--- libpod-1.4.0.orig/vendor/github.com/containers/image/signature/mechanism_gpgme.go
+++ libpod-1.4.0/vendor/github.com/containers/image/signature/mechanism_gpgme.go
@@ -8,7 +8,7 @@ import (
 	"io/ioutil"
 	"os"
 
-	"github.com/mtrmac/gpgme"
+	"github.com/proglottis/gpgme"
 )
 
 // A GPG/OpenPGP signing mechanism, implemented using gpgme.
