Last-Update: 2019-09-09
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: update library namespace

--- a/libpod/boltdb_state.go
+++ b/libpod/boltdb_state.go
@@ -6,9 +6,9 @@
 	"strings"
 	"sync"
 
 	"github.com/containers/libpod/libpod/define"
-	bolt "github.com/etcd-io/bbolt"
+	bolt "github.com/coreos/bbolt"
 	jsoniter "github.com/json-iterator/go"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
--- a/libpod/boltdb_state_internal.go
+++ b/libpod/boltdb_state_internal.go
@@ -8,9 +8,9 @@
 
 	"github.com/containers/libpod/libpod/define"
 	"github.com/containers/libpod/pkg/rootless"
 	"github.com/containers/storage"
-	bolt "github.com/etcd-io/bbolt"
+	bolt "github.com/coreos/bbolt"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
