Index: libpod/pkg/bindings/generator/generator.go
===================================================================
--- libpod.orig/pkg/bindings/generator/generator.go
+++ libpod/pkg/bindings/generator/generator.go
@@ -169,7 +169,7 @@ func main() {
 		}
 
 		// go import file
-		goimport := exec.Command("../../../test/tools/build/goimports", "-w", out.Name())
+		goimport := exec.Command("goimports", "-w", out.Name())
 		goimport.Stderr = os.Stdout
 		if err := goimport.Run(); err != nil {
 			fmt.Println(err)
