From: Michael Biebl <biebl@debian.org>
Subject: Please don't enable podman-auto-update.service in default.target
Date: Tue, 26 Jan 2021 11:46:06 +0100
Bugs-Debian: https://bugs.debian.org/981097

--- a/contrib/systemd/auto-update/podman-auto-update.service
+++ b/contrib/systemd/auto-update/podman-auto-update.service
@@ -10,4 +10,4 @@
 ExecStartPost=/usr/bin/podman image prune -f
 
 [Install]
-WantedBy=multi-user.target default.target
+WantedBy=multi-user.target
