Last-Update: 2019-06-17
Forwarded: https://github.com/containers/libpod/issues/3346
Bug-Upstream: https://github.com/containers/libpod/issues/3346
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: build compilation failure with newer Logrus

--- a/cmd/podman/export.go
+++ b/cmd/podman/export.go
@@ -6,9 +6,9 @@
 	"github.com/containers/libpod/cmd/podman/cliconfig"
 	"github.com/containers/libpod/cmd/podman/shared/parse"
 	"github.com/containers/libpod/pkg/adapter"
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
+	"golang.org/x/crypto/ssh/terminal"
 	"github.com/spf13/cobra"
 )
 
 var (
@@ -61,9 +61,9 @@
 	}
 
 	if len(output) == 0 {
 		file := os.Stdout
-		if logrus.IsTerminal(file) {
+		if terminal.IsTerminal(int(file.Fd())) {
 			return errors.Errorf("refusing to export to terminal. Use -o flag or redirect")
 		}
 		output = "/dev/stdout"
 	}
--- a/cmd/podman/save.go
+++ b/cmd/podman/save.go
@@ -8,9 +8,9 @@
 	"github.com/containers/libpod/cmd/podman/shared/parse"
 	"github.com/containers/libpod/pkg/adapter"
 	"github.com/containers/libpod/pkg/util"
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
+	"golang.org/x/crypto/ssh/terminal"
 	"github.com/spf13/cobra"
 )
 
 const (
@@ -81,9 +81,9 @@
 	}
 
 	if len(c.Output) == 0 {
 		fi := os.Stdout
-		if logrus.IsTerminal(fi) {
+		if terminal.IsTerminal(int(fi.Fd())) {
 			return errors.Errorf("refusing to save to terminal. Use -o flag or redirect")
 		}
 		c.Output = "/dev/stdout"
 	}
